From bd40bc76f35fdb65d878ba1d6a2239cafd58ca8a Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Mon, 16 Jan 2017 13:05:14 +0200
Subject: [PATCH 0729/1345] fix: mmc: a70x0: a80x0: remove no-1-8-v flag from
 eMMC/SD node

commit  f34e457a56eb3c31a0c1cc372632a55479446af7 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Remove no-1-8-v flag from eMMC/SD node, both Armada-70x0/80x0 DB
board can work with eMMC 1.8v.

Change-Id: I61551c7b5158fc3ec349048050e769e52a64bfb9
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35623
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35677
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-db.dtsi |    1 -
 arch/arm64/boot/dts/marvell/armada-8040-db.dtsi |    1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
index 408747b..6d2a98d 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-db.dtsi
@@ -110,7 +110,6 @@
 			};
 			sdhci@780000 {
 				bus-width = <4>;
-				no-1-8-v;
 				non-removable;
 			};
 			spi@700680 {
diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi b/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
index 90ac54d..55ab0f6 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dtsi
@@ -117,7 +117,6 @@
 			};
 			sdhci@780000 {
 				bus-width = <4>;
-				no-1-8-v;
 				non-removable;
 			};
 		};
-- 
1.7.9.5

